.idea/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Default ignored files
```

Comment:
this file was not removed by gitignore, so you must remove .idea/** gitignore entry, remove all files from .idea folder from index, and them add them again to gitignore, to remove them from repository